src/app/app.component.ts

```diff
@@ -9,219 +9,6 @@ import { Component, OnInit } from '@angular/core';
 export class AppComponent implements OnInit {
 
   constructor() { }
-
-  successMessage: string = '';
-  errorMessage: string = '';
-  receiptLink: string = '';
-
-  currencies = [ 'usd', 'aud', 'eur', 'gbp', 'krw', 'nzd', 'thb', 'jpy', 'brl', 'cad' ];
-
-  currencySymbols = {
-    'usd': '$',
-    'aud': '$',
-    'eur': '€',
-    'gbp': '£',
-    'krw': '₩',
-    'nzd': '$',
-    'thb': '฿',
-    'jpy': '¥',
-    'brl': '$',
-    'cad': '$'
-  };
-
-  model: AppModel = new AppModel('usd', 'single', 0);
-
-  amounts: Array<string>;
-
-  private stripe;
-
-  constructor(private http: HttpClient) {
-    const self = this;
-    let stripe_key = 'pk_test_eM74cMSAfGXWvrLZnz7gCXuA';
-    if ((<any>window).app_data) {
-      this.model.currency = (<any>window).app_data.currency;
-      stripe_key = (<any>window).app_data.stripe_key;
-    }
-    this.updateAmounts();
-    this.stripe = (<any>window).StripeCheckout.configure({
-      key: stripe_key,
-      image: 'https://keyman.com/cdn/dev/img/icon1.png',
-      locale: 'auto',
-      billingAddress: true,
-      token: function(token, tokenData) {
-        console.log('got token', token, 'data ', tokenData);
-        // You can access the token ID with `token.id`.
-        // Get the token ID to your server-side code for use.
-        const postData = {
-          'token': token,
-          'amount': self.amountForCard(),
-          'currency': self.model.currency,
-          'data': tokenData,
-        }; 
-        const apiURL = 'https://donate.keyman.com/api/charge';
-
-        self.http.post(apiURL, postData, {
-          headers: new HttpHeaders().set('Content-Type', 'application/json'),
-          observe: 'response', // TODO How do we catch 4xx errors?
-        }).subscribe(
-          success => {
-            // Handle result
-            console.log('POST call successful value returned in body', success);
-            self.successMessage = 'The transaction has been completed. Thank you for you donation!';
```

Comment:
@mcdurdin here is some the missing code from the merge. This occurred because we changed file names.